contrib/build.Dockerfile

```diff
@@ -27,7 +27,7 @@ RUN apt-get install -y \
     libudev-dev \
     faketime \
     zip \
-    pyqt5-dev \
+    python3-pyqt6 \
```

Comment:
In 219aa6f4ee94dee5721cd6d27bf35ed0299fa311 "Bump pyqt5 to pyqt6"

Note that this isn't used for speculos, but rather for our own gui build.

Reply:
Ah, I'll revert that. We can upgrade the gui later.